pkg/cluster/install.go

```diff
@@ -62,6 +63,12 @@ func (m *manager) adminUpdate() []steps.Step {
 		)
 	}
 
+	if isEtcdRecovery || isEverything {
```

Comment:
Initial reaction - I think this should be moved out of the code path of a normal PUCM and we should have `isEtcdRecovery` be something we intentionally trigger, after deliberation on our end. I see that your `fixEtcd` code path checks etcd operator for a degraded status, but I think it's risky to rely on that during a normal PUCM run. We'd want to do this very intentionally

Reply:
I do feel confident how the degraded status is verified, but I also agree it should be a very intentional action. I've updated `install.go` to only run explicitly.